network/Cargo.toml

```diff
@@ -38,10 +38,10 @@ libra-types = { path = "../types", version = "0.1.0" }
 libra-workspace-hack = { path = "../common/workspace-hack", version = "0.1.0" }
 memsocket = { path = "memsocket", version = "0.1.0" }
 netcore = { path = "netcore", version = "0.1.0" }
-noise = { path = "noise", version = "0.1.0" }
 num-variants = { path = "../common/num-variants", version = "0.1.0" }
 proptest = { version = "0.10.0", default-features = true, optional = true }
 stream-ratelimiter = { path = "../common/stream-ratelimiter", version = "0.1.0" }
+rand_core = { version = "0.5.1", optional = true }
```

Comment:
I would have expected to see this one in `[dev-dependencies]`?

Reply:
fuzzing is not run as dev : o